packages/components/src/hooks/useClickCapture.ts

```diff
@@ -0,0 +1,23 @@
+import { useState, useRef, useEffect } from 'react';
+
+const useClickCapture = (triggerTimeout = 0) => {
+  const [clicked, setClicked] = useState(false);
+  const clickCaptureRef = useRef<HTMLDivElement>(null);
```

Comment:
Good point! Clicks are being tracked with basic eventListeners so `div` was something they can be added to, and that has `ref` attribute, so I figured they're good for this wrapper use. 

But I guess any element that has these would be OK, I maybe need to look into this on the next PR, since there might be cases still where div wrapper will break something..